### DISCUSSION TITLE
How to change api.ts code?

### DISCUSSION THREAD

**Discussion by da-da-da** [nocodb/nocodb#7245] [Q&A]
I made some changes to api.ts in the nocodb-sdk, but when I run the project, my modifications are not included. The code has changed back to the original content. Could you please tell me how to build the nocodb-sdk?

**Comment by wingkwong** [comment]
`pnpm run build` in sdk then run install again on both frontend and backend. Alternatively, you can do the same action by running  `pnpm run bootstrap` in the project root directory.
